pkg/api/v1alpha1/cluster_types.go

```diff
@@ -253,6 +270,27 @@ func WorkerNodeGroupConfigurationSliceTaintsEqual(a, b []WorkerNodeGroupConfigur
 	return true
 }
 
+func WorkerNodeGroupConfigurationsLabelsMapEqual(a, b []WorkerNodeGroupConfiguration) bool {
+	m := make(map[string]map[string]string, len(a))
+	for _, nodeGroup := range a {
+		m[nodeGroup.Name] = nodeGroup.Labels
+	}
+
+	for _, nodeGroup := range b {
+		if _, ok := m[nodeGroup.Name]; !ok {
+			// this method is not concerned with added/removed node groups,
+			// only with the comparison of taints on existing node groups
```

Comment:
```suggestion
			// only with the comparison of nodes on existing node groups
```

I thought I recognized this code :)

Reply:
Haha yup